site/app/views/grading/ElectronicGraderView.php

```diff
@@ -1172,7 +1172,7 @@ public function renderGradingPanelHeader(bool $isPeerPanel, bool $isStudentInfoP
 
     /**
      * Render the Autograding Testcases panel
-     * @param AutoGradedVersion $version_instance
+     * @param AutoGradedVersion|null $version_instance
```

Comment:
It may be good to add the typing in the function definition also if it's being updated here (for a nullable AutoGradedVersion it would be `AutoGradedVersion? $version_instance`)